docs/stdlib/numbers.rst

```diff
@@ -740,10 +762,8 @@ from :eql:type:`str` and :eql:type:`json`.
 
     :index: parse bigint
 
-    Create a :eql:type:`bigint` value.
-
-    Parse a :eql:type:`bigint` from the input *s* and optional format
-    specification *fmt*.
+    Returns a :eql:type:`bigint` value from ``s`` with a possible format of
+    ``fmt``:
 
     .. code-block:: edgeql-repl
 
```

Comment:
If you accept the above suggestion for lines 765-766, you can remove 774-776 since that information is already encapsulated in the new detail paragraph.